src/main/java/com/futurecollars/OOP2Task7/DayName.java

```diff
@@ -0,0 +1,32 @@
+package com.futurecollars.OOP2Task7;
+
+public class DayName {
+    private String day;
+
+    public String getName(Day day) {
+        switch (day) {
+            case MONDAY:
+                this.day = "Monday";
+                break;
+            case TUESDAY:
+                this.day = "Tuesday";
+                break;
+            case WEDNESDAY:
+                this.day = "Wednesday";
+                break;
+            case THURSDAY:
+                this.day = "Thursday";
+                break;
+            case FRIDAY:
+                this.day = "friday";
+                break;
+            case SATURDAY:
+                this.day = "Saturday";
+                break;
+            default:
+                this.day = "Sunday";
+                break;
+        }
+        return this.day;
```

Comment:
przekombinowane ;) 
Zamiast przypisywać dzień to 'this.day', aby później tylko to zwrocić - po prostu zwróć tę wartość. Po co ta zmienna i to jeszcze w klasie, a nie lokalna. :P